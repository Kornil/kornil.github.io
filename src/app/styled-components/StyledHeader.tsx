import styled from "react-emotion";

const StyledHeader = styled.header`
  grid-area: nav;
  padding: 8px 0;
  height: 40px;
  display: flex;
  align-items: center;
  justify-content: center;
`;

export default StyledHeader;
