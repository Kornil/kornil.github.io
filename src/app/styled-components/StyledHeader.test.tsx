import { shallow } from "enzyme";
import React from "react";

import StyledHeader from "./StyledHeader";

test("it works", () => {
  const wrapper = shallow(<StyledHeader />);
  expect(wrapper).toMatchSnapshot();
});
