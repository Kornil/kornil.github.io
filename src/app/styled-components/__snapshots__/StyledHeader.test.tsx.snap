// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.emotion-0 {
  padding: 8px 0;
  height: 40px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<header
  className="emotion-0 emotion-1"
/>
`;
